libsweep/include/protocol.h

```diff
@@ -6,7 +6,7 @@
  * Implementation detail; not exported.
  */
 
-#include <stdint.h>
+#include <cstdint>
```

Comment:
@daniel-j-h: I think you are right <cstdint> only guarantees the availability of `std::int32_t` not the un-prefixed version.